src/posit/connect/client.py

```diff
@@ -50,6 +50,9 @@ def connect_version(self):
     def me(self) -> User:
         url = urls.append_path(self.config.url, "v1/user")
         response = self.session.get(url)
+        body = response.json()
+        guid = body["guid"]
+        url = urls.append_path(self.config.url, f"v1/users/{guid}")
```

Comment:
Is this necessary because you can't `PUT v1/users/`? Should we add a test covering this behavior?

Reply:
Correct. Yes, we should add some test coverage for this endpoint.